library/tasks/HISAT2.wdl

```diff
@@ -8,7 +8,7 @@ task HISAT2PairedEnd {
 
   # runtime values
   String docker = "quay.io/humancellatlas/secondary-analysis-hisat2:v0.2.2-2-2.1.0"
-  Int machine_mem_mb = 5000
+  Int machine_mem_mb = 15000
```

Comment:
why do you request `15G` ?

Reply:
These tasks request 4 cores and 5GB of memory. On GCP, this should get you an n1-standard-4, which actually has 15GB of memory. On AWS, you might get a c4.xlarge, which has 7.5GB of memory.

Some of the tasks appear to actually require between 7.5 and 15GB of memory, so they succeed on GCP but fail on AWS even though both platforms provide an instance that meets the declared requirements.

Reply:
Yes, it is fair. But 15G is actually overkill for this task. On average, HISAT2 uses ~8G so I would like to request 10G instead 15G here. Does `10G` work with AWS?

Reply:
It does, but just like GCP you'll get an instance with 15 or 16GB of memory. I'm a little reluctant to set it lower than what we're actually using in production. If 8GB is average but say 10% of jobs use >10GB, then we'd never notice problems in the Green Box, but other users would see lots of failures.